docs/python_api/datasource/_datasource.md

```diff
@@ -4,4 +4,29 @@ title: Data Source
 
 # Data Source
 
-TBC
+Open Targets Genetics is designed to be scalable and readily extensible. New data sources will continually be curated, reviewed, incorporated, and documented here and in the release notes. Please get in touch with any suggestions.
+
+This section contains information about the data sources used in Open Targets Genetics.
```

Comment:
I would consider the pipeline, hence the documentation of the pipeline and the OT Genetics as a data product two slightly different entities. It's true that the pipeline feeds into the OTGenetics, however making the link too thight could make things complicated: imagine a scenario, where the pipeline is updated for a new datasource, or a new version of something, the new software release is out, but OTG become off sync. 

The point is clearer when thinking about methods: the pipeline might have more methods implemented than actually applied on the data for OTG. We can decide to retire datasets, but their parsers migth stay if our partners still want to integrate on their instances. 

I know, this is quite nuanced. What do you think?

Reply:
I think, deleting the first sentence ("Open Targets Genetics is designed to be scalable and readily extensible. New data sources will continually be curated, reviewed, incorporated, and documented here and in the release notes. Please get in touch with any suggestions.") helps to avoid the confusion.